src/ol_infrastructure/infrastructure/consul/__main__.py

```diff
@@ -219,18 +220,80 @@
         "private_ip": consul_instance.private_ip,
         "ipv6_address": consul_instance.ipv6_addresses,
     }
+######
 
 fifteen_minutes = 60 * 15
 consul_domain = route53.Record(
     f"consul-{environment_name}-dns-record",
-    name=f"consul-{salt_environment}.odl.mit.edu",
+    # name=f"consul-{salt_environment}.odl.mit.edu",
```

Comment:
We still want to set up a DNS record with the name scoped to the environment, but we no longer need to use the salt environment as the canonical name. This should template in the `environent_name` variable

Reply:
On further consideration, we won't be able to use the instance addresses as the destination for the DNS record since they will be managed by the ASG. We may need to set up a load balancer to handle routing to the instances for API calls.